src/test/java/com/trustly/api/RequestsTest.java

```diff
@@ -0,0 +1,357 @@
+import com.trustly.api.domain.methods.cancelcharge.CancelChargeRequestData;
+import com.trustly.api.domain.methods.charge.ChargeRequestData;
+import com.trustly.api.domain.methods.charge.ChargeRequestDataAttributes;
+import com.trustly.api.domain.methods.deposit.DepositRequestData;
+import com.trustly.api.domain.methods.deposit.DepositRequestDataAttributes;
+import com.trustly.api.domain.methods.deposit.DepositResponseData;
+import com.trustly.api.domain.methods.refund.RefundRequestData;
+import com.trustly.api.domain.methods.refund.RefundRequestDataAttributes;
+import com.trustly.api.domain.methods.registeraccount.RegisterAccountRequestData;
+import com.trustly.api.domain.methods.registeraccount.RegisterAccountRequestDataAttributes;
+import com.trustly.api.domain.methods.registeraccount.RegisterAccountResponseData;
+import com.trustly.api.domain.methods.registeraccountpayout.RegisterAccountPayoutRequestData;
+import com.trustly.api.domain.methods.registeraccountpayout.RegisterAccountPayoutRequestDataAttributes;
+import com.trustly.api.domain.methods.registeraccountpayout.RegisterAccountPayoutResponseData;
+import com.trustly.api.domain.methods.selectaccount.SelectAccountRequestData;
+import com.trustly.api.domain.methods.selectaccount.SelectAccountRequestDataAttributes;
+import com.trustly.api.domain.methods.selectaccount.SelectAccountResponseData;
+import com.trustly.api.domain.methods.withdraw.WithdrawRequestData;
+import com.trustly.api.domain.methods.withdraw.WithdrawRequestDataAttributes;
+import com.trustly.api.domain.methods.withdraw.WithdrawResponseData;
+import java.util.UUID;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * These test will only work if a local environment exists. At the USER_HOME root these files is needed: trustly_client_username.txt - your
+ * username trustly_client_password.txt - your password trustly_client_public.pem - your public key trustly_client_public.pem - your private
+ * key
+ * <p>
+ * If those files exist and is correct you are able to make requests to Trustlys test environment
+ */
+class RequestsTest {
+
+  private final TrustlyApiClientSettings settings = TrustlyApiClientSettings.forTest()
+    .withCredentialsFromUserHome()
+    .withCertificatesFromUserHome()
+    .andTrustlyCertificate();
+
+  @Test
+  void testDeposit() throws Exception {
+
+    try (TrustlyApiClient client = new TrustlyApiClient(settings)) {
+
+      DepositRequestData request = DepositRequestData.builder()
+        .notificationUrl("https://fake.test.notification.trustly.com")
+        .endUserId("pontus.eliason@trustly.com")
+        .messageId(UUID.randomUUID().toString())
+        .attributes(
+          DepositRequestDataAttributes.builder()
+            .currency("EUR")
+            .amount("100.00")
+            .firstname("John")
+            .lastname("Doe")
+            .email("pontus.eliason@trustly.com")
```

Comment:
Is there a reason to not use John.Doe@trustly.com?

Reply:
Nope :) Will fix